adapter_pipelines/submit.wdl

```diff
@@ -25,11 +26,10 @@ task get_metadata {
 
     get-analysis-workflow-metadata \
       --analysis_output_path ${analysis_output_path} \
-      --cromwell_url ${cromwell_url} \
-      --use_caas ${use_caas}
```

Comment:
Since removing the Cromwell-credentials docker image forces us to drop support of running Adapter workflows in any non-OAuth Cromwell instances, I think we need to remove the `use_caas ` flag everywhere across the WDLs(e.g. `Boolean use_caas`) and Python code. 

Besides, could you add something like "The adapter workflows can only run in OAuth-enabled Cromwell instances" explicitly in the Readme?

Reply:
I tried to be even more specific and added that the workflows can only run in Cromwell instances that are using SAM for Identity Access Management

Reply:
> I think we need to remove the use_caas flag everywhere across the WDLs(e.g. Boolean use_caas) and Python code.

@rexwangcc do you think we should remove it from the lira config as well? Or just the part that is creating the adapter workflow inputs?

Reply:
Maybe the part that is creating the adapter workflow inputs?